priv/behaviour/LICENSE

```diff
@@ -0,0 +1,191 @@
+                                 Apache License
```

Comment:
Any reason why the `priv` directory was preferred to the `_SUITE_data` directory in this case?

Reply:
And the reason it is in priv is that it deliberately has warnings in it, to confirm we can get diagnostics when there is a behaviour. So it breaks the "compile clean" rule for the main stuff.  So it is the same reason as the other stuff currently in priv.